AppServices/Helpers/CustomerProfile.cs

```diff
@@ -0,0 +1,18 @@
+﻿using APIDesafioWarren.Models;
+using App.Services;
+using Application.Models.DTOs;
+using AutoMapper;
+using System.Collections.Generic;
+
+namespace AppServices.Helpers
```

Comment:
```suggestion
namespace AppServices.Profiles
```
Renomear a pasta para Profiles